src/formatISO/index.js

```diff
@@ -0,0 +1,111 @@
+import toDate from '../toDate/index.js'
+import isValid from '../isValid/index.js'
+import addLeadingZeros from '../_lib/addLeadingZeros/index.js'
+
+/**
+ * @name formatISO
+ * @category Common Helpers
+ * @summary Format the date according to the ISO 8601 standard (http://support.sas.com/documentation/cdl/en/lrdict/64316/HTML/default/viewer.htm#a003169814.htm).
+ *
+ * @description
+ * Return the formatted date string in ISO 8601 format. Options may be passed to control the parts and notations of the date.
+ *
+ * @param {Date|Number} date - the original date
+ * @param {Object} [options] - an object with options.
+ * @param {'extended'|'basic'} [options.format='extended'] - if 'basic', doesn't show delimeters between date and time values.
```

Comment:
```suggestion
 * @param {'extended'|'basic'} [options.format='extended'] - if 'basic', hide delimiters between date and time values.
```
sorry for nitpicking here, just a suggestion for changes:

1. "doesn't show" --> "hide"
2. "delimeter" --> "delimiter"

Reply:
Thanks a lot! 🙏